README.yaml

```
@@ -37,6 +37,11 @@ references:
 description: |-
   Collection of Makefiles and test scripts to facilitate testing Terraform modules, Kubernetes resources, Helm charts, and more.
 
+  ## Prerequisites
+
+  1. [Bats-core](https://github.com/bats-core/bats-core)
+  1. Bash v5+
```

Our test harness uses 4 I believe. Didn't even know v5 was available.
```suggestion
  1. Bash v4+
```

---

But hey - let's keep it

---

@osterman it works w/ v5 AFAICT 😄